packages/fury-adapter-oas3-parser/lib/parser/oas/parseParameterObject.js

```diff
@@ -95,6 +95,7 @@ function parseParameterObject(context, object) {
     [hasKey('in'), parseIn],
     [hasKey('description'), parseString(context, name, false)],
     [hasKey('required'), parseBoolean(context, name, false)],
+    [hasKey('example'), parseString(context, name, false)],
```

Comment:
According to the [OAS3 spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameter-object) ANY element can by an example, and it (only) SHOULD match the schema; I think we can just substitute `parseString(context, name, false)` with the identity function (`e => e`) here.

Reply:
Changed